### ISSUE TITLE
[BUG] white screen

### ISSUE THREAD

**Issue by Braunfeltd** [Bin-Huang/chatbox#1117]
**Bug Description**
When running the app seems to lock up and go to a white screen which forces me to close and open again to use
**Steps to Reproduce**
Please provide the steps to reproduce the bug:

its random.   I could be just sitting here and than soon as I click to enter it goes white.   

some things that may contribute is that I am using graphical programs in the background that use the gpu.

**Desktop (please complete the following information):**

-   Operating System: windows 11
-   Application Version: 1.2.2

not much else I can tell you nothing special about it just the whole app goes white.

**Comment by TridentDoubleMint** [comment]
Having the same issue. Also my PC workin overtime after it happens until i close it.

**Comment by alex-tarasov-6669** [comment]
have same issue many many during last many version of tool.. it generate answer and in some moment freeze, than app screen became white and nothing happens, only restart.

**Comment by puzhiyu123** [comment]
Have the same issue here.

**Comment by rodion-m** [comment]
Also became to reproduce in the latest version (v1.3.17). I think, it happened after in my chat history appeared messages with a long context (it's just in the history, but perhaps for some reason it may affect other chats?). 

@Bin-Huang Need your attention, cause it's a blocking bug that prevents me from using Chatbox.

<img width="1376" alt="image" src="https://github.com/user-attachments/assets/1d5874ff-e522-4dd9-bc14-9cc1b2618b8b">

**Comment by Bin-Huang** [comment]
> Also became to reproduce in the latest version (v1.3.17) when using Sonnet 3.5. I think it happened after in my chat history appeared messages with a long context (it's just in the history, but perhaps for some reason it may affect other chats?).
> 
> @Bin-Huang Need your attention, cause it's a blocking bug that prevents me from using Chatbox.
> 
> <img alt="image" width="1376" src="https://private-user-images.githubusercontent.com/36400912/356803716-1d5874ff-e522-4dd9-bc14-9cc1b2618b8b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzNjA4NjMsIm5iZiI6MTcyMzM2MDU2MywicGF0aCI6Ii8zNjQwMDkxMi8zNTY4MDM3MTYtMWQ1ODc0ZmYtZTUyMi00ZGQ5LWJjMTQtOWNjMWIyNjE4YjhiLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDA3MTYwM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWEzNGRmODZjNTQwZDkwNDM5Y2M2MjczOTc0ODE1NGI3MWU4MDdhYjJhNGM4ZTc0YWU4MWFhY2U2ZTlmZDE4MWQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3nqrce6do0ic8Rg1BTGxX-sCvmlsaKQaemhdk9hq4vg">

What operating system and version are you using? This info will help me better replicate the issue locally.

**Comment by rodion-m** [comment]
> What operating system and version are you using? This info will help me better replicate the issue locally.

Apple M3 Pro 18 gb, Sonoma 14.5 (23F79). Also, I have at least 100 chat histories on the left side (some of them had around 500k tokens).

**Comment by rodion-m** [comment]
It's still not fixed. It's a blocker of usage, had to switch to another app. Sadly.

**Comment by Bin-Huang** [comment]
This is indeed a strange issue and I still have no clue what's causing it. I hope more people can share more details about their experience. Until I figure out the root cause, please use the Chatbox Web version as a temporary solution - it should work fine there: https://web.chatboxai.app/

**Comment by rodion-m** [comment]
> This is indeed a strange issue

Looks like a base technology issue. What do you use? Electron? Have you tried to search the base technology-related issues?

**Comment by Bin-Huang** [comment]
@rodion-m  Yes, you're right - it does look like that kind of issue. I'm using Electron (v22) and found some discussions about similar appening with certain GPUs. I'll keep looking into this, but since I'm working on several new features right now, it might take some time to finvestigate the issue fully.

**Comment by rodion-m** [comment]
Ok, thank you. Also, consider linking related issues into this thread (just mention them).

**Comment by Bin-Huang** [comment]
@rodion-m I'm wondering if the issue persists even after restarting the app. If so, here are a couple of things you can try:

1. Try rolling back to a previous backup. Open Terminal and run `cd 'Library/Application Support/xyz.chatboxapp.app' && ls`. You'll see some backup files named like `config-backup-2024-8-4.json`. Just use `mv config-backup-2024-8-4.json config.json` to restore to an older version. Then restart the app and see if it works. Feel free to try different backup files.

2. Or you can start fresh by clearing the app data. In Terminal, run `cd 'Library/Application Support/xyz.chatboxapp.app' && rm config.json` - this will reset the app to its initial state, which often fixes things.

Let me know how it goes after you try these steps! This will help me better understand what might be causing the issue. 😊
